charactersheet/charactersheet/templates/dm/dmscreen.tmpl.html

```diff
@@ -0,0 +1,350 @@
+              <li>The creature automatically fails Strength and Dexterity saving throws.</li>
+              <li>The creature has resistance to all damage.</li>
+              <li>The creature is immune to poison and disease, although a poison or disease already in  its system is suspended, not neutralized.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>Poisoned</td>
+          <td>
+            <ul>
+              <li>A poisoned creature has disadvantage on attack rolls and ability checks.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>Prone</td>
+          <td>
+            <ul>
+              <li>A prone creature’s only movement option is to crawl, unless it stands up and thereby ends the condition.</li>
+              <li>The creature has disadvantage on attack rolls.</li>
+              <li>An attack roll against the creature has advantage if the attacker is within 5 feet of  the creature. Otherwise, the attack roll has disadvantage.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>Restrained</td>
+          <td>
+            <ul>
+              <li>A restrained creature’s speed becomes 0, and it can’t benefit from any bonus to its speed.</li>
+              <li>Attack rolls against the creature have advantage, and the creature’s attack rolls have  disadvantage.</li>
+              <li>The creature has disadvantage on Dexterity saving throws.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>Stunned</td>
+          <td>
+            <ul>
+              <li>A stunned creature is incapacitated (see the condition), can’t move, and can speak only falteringly.</li>
+              <li>The creature automatically fails Strength and Dexterity saving throws.</li>
+              <li>Attack rolls against the creature have advantage.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>Unconscious</td>
+          <td>
+            <ul>
+              <li>An unconscious creature is incapacitated (see the condition), can’t move or speak, and is unaware of its surroundings</li>
+              <li>The creature drops whatever it’s holding and falls prone.</li>
+              <li>The creature automatically fails Strength and Dexterity saving throws.</li>
+              <li>Attack rolls against the creature have advantage.</li>
+              <li>Any attack that hits the creature is a critical hit if the attacker is within 5 feet of  the creature.</li>
+            </ul>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12">
+    <table class="table table-responsive table-ac-bordered">
+    <h3>Travel Pace<br />
+      <small>While traveling, a group of adventurers can move at a normal, fast, or  slow  pace, as  shown on the Travel Pace table.</small>
```

Comment:
Can this change from

>...or  slow  pace, as  shown on the Travel Pace table

to

>...or  slow  pace, as  shown here.

Also, there's two spaces between a lot of the words in this sentence. Can we get rid of those?